bpython/test/test_import_not_cyclical.py

```diff
@@ -0,0 +1,108 @@
+from bpython.test import unittest
+from bpython.importcompletion import find_modules
+import os, sys, tempfile
+
+
+class TestAvoidSymbolicLinks(unittest.TestCase):
+    def setUp(self):
+        with tempfile.TemporaryDirectory() as import_test_folder:
+            try:
+                os.chdir(import_test_folder)
```

Comment:
Are these `chdir`s really necessary? All the paths that are calculated based on `getcwd` seem to be computable from `import_test_folder`.

Reply:
I am using the chdir to make the __init__.py files in the right folder, is there a better way?

Reply:
Yes, you can build the paths with `os.path.join` and then pass it to `open`. For example: `open(os.path.join(import_test_folder, "Level0", "__init__.py"), "w")`. The same can be done for the `os.mkdir` calls.